demo/src/lt.tooltip.js

```diff
@@ -64,13 +64,42 @@ angular.module('lt.tooltip', [])
         }
     };
 
+    var positionArrow = function($arrow, direction) {
```

Comment:
You may consider breaking this up into getArrowPosition(direction) and positionArrow($arrow, direction) functions. I'm on the fence about this because it's such a small function to begin with, but logically it is doing two things. Up to you if you want to split them up.